src/vivarium_census_prl_synth_pop/results_processing/addresses.py

```diff
@@ -43,23 +43,36 @@ def get_address_id_maps(
     A dictionary of pd.Series suitable for pd.Series.map, indexed by `address_id`
 
     """
-    if column_name != "address_id":
-        raise ValueError(f"Expected `address_id`, got `{column_name}`")
+
+    if column_name == "address_id":
+        output_cols_superset = [column_name, "state_id", "state", "puma", "po_box"]
+    elif column_name == "employer_address_id":
+        output_cols_superset = [
+            column_name,
+            "employer_state_id",
+            "employer_state",
+            "employer_puma",
+        ]
+    else:
+        raise ValueError(
+            f"Expected `address_id` or 'employer_address_id', got `{column_name}`"
+        )
+
```

Comment:
A way to avoid the `if/elif` is to try/except a dictionary lookup. I've seen it/used it a bunch in artifact code.   See for example: https://github.com/ihmeuw/vivarium_gates_iv_iron/blob/862b057457cea2410e8d24c2c9c38c43419cd8b2/src/vivarium_gates_iv_iron/data/loader.py#L464-L472